pages/k8s/cni-calico.md

```diff
@@ -63,23 +63,8 @@ that would apply to `charmed-kubernetes` to this bundle also.
-| calico-policy-image | string | docker.io/calico/kube-controllers:v3.6.1 | The image id to use for calico/kube-controllers                |
-| cidr                | string | 192.168.0.0/16                           | Network CIDR assigned to Calico. This is applied to the default Calico pool, and is also communicated to the Kubernetes charms for use in kube-proxy configuration. |
-| global-as-number    | int    | 64512                                    | Global AS number.
-| global-bgp-peers    | string | []                                       | List of global BGP peers. Each BGP peer is specified with an address and an as-number. |
-| ipip                | string | Never                                    | IPIP encapsulation mode. Must be one of "Always", "CrossSubnet", or "Never". This is incompatible with VXLAN encapsulation. If VXLAN encapsulation is enabled, then this must be set to "Never". |
-| manage-pools        | bool   | True                                     | If true, a default pool is created using the cidr and ipip charm configuration values. Warning: When manage-pools is enabled, the charm will delete any pools that are unrecognized. |
-| nat-outgoing        | bool   | True                                     | Enable NAT on outgoing traffic                                 |
-| node-to-node-mesh   | bool   | True                                     | When enabled, each Calico node will peer with every other Calico node in the cluster. |
-| route-reflector-cluster-ids | string | {}                               | Mapping of unit IDs to route reflector cluster IDs. Assigning a route reflector cluster ID allows the node to function as a route reflector. |
-| subnet-as-numbers   | string | {}                                       | Mapping of subnets to AS numbers, specified as YAML. Each Calico node will be assigned an AS number based on the entries in this mapping. |
-| subnet-bgp-peers    | string | {}                                       | Mapping of subnets to lists of BGP peers. Each BGP peer is specified with an address and an as-number. |
-| unit-as-numbers     | string | {}                                       | Mapping of unit IDs to AS numbers, specified as YAML. Each Calico node will be assigned an AS number based on the entries in this mapping. |
-| unit-bgp-peers      | string | {}                                       | Mapping of unit IDs to lists of BGP peers. Each BGP peer is specified with an address and an as-number. |
-| vxlan               | string | Never                                    | VXLAN encapsulation mode. Must be one of "Always", "CrossSubnet", or "Never". This is incompatible with IPIP encapsulation. If IPIP encapsulation is enabled, then this must be set to "Never". |
+A full list of Calico configuration options and their descriptions can be found
+in the [Calico charm] page.
```

Comment:
@evilnick I noticed this table was missing a couple options (ignore-loose-rpf, disable-vxlan-tx-checksumming). I thought it might make sense to have this reference the auto-generated charm page instead, which is more likely to stay up-to-date. If you disagree, let me know and I'll revert this and add the missing options.

Reply:
Well, now that I look, those config options are missing from the charm page too. What's the process for updating those?

Reply:
I will regenerate them manually - when you say charm page, you mean the store page? Those are supposed to be auto-generated by the store

Reply:
I meant this page: https://ubuntu.com/kubernetes/docs/charm-calico